guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -3,6 +3,9 @@
 
 [id="foreman-headline-features"]
 == Headline Features
+=== Foreman Ansible plugin
+* A new option to enable the callback plugin for each ansible job template was added.
+* When an ansible job execution results in an error, display the error message in the config report UI.
```

Comment:
We should capitalize Ansible
```suggestion
* A new option to enable the callback plugin for each Ansible job template was added.
* When an Ansible job execution results in an error, display the error message in the config report UI.
```

But we always write headline features for humans. That means a paragraph or two of text, rather than a list. Feel free to use a bit more space to explain why it's nice.